src/components/product/ProductSummary/product-summary.scss

```diff
@@ -0,0 +1,78 @@
+.product-summary[data-store-card] {
+  position: relative;
```

Comment:
What about adding a transition here the make the hover smooth? We could use `transition: box-shadow .5s ease;`